Source/Core/Core/Core.cpp

```diff
@@ -125,6 +129,13 @@ void FrameUpdateOnCPUThread()
     NetPlay::NetPlayClient::SendTimeBase();
 }
 
+void FrameAdvance()
```

Comment:
The name of this function makes it sound like it makes the emulation run for one frame. Maybe something like `OnFrameEnd` would be better?

Reply:
I've changed it to OnFrameEnd.